core/src/main/java/org/jboss/wise/core/client/impl/reflection/BasicWSDynamicClientImpl.java

```diff
@@ -169,7 +169,7 @@ private synchronized void initClassLoader(File outputDir) throws WiseRuntimeExce
 			servicesMap.put(annotation.name(), service);
 		    }
 		} catch (Exception e) {
-		    e.printStackTrace();
+		    //e.printStackTrace();
```

Comment:
Might make sense to keep logging the full exception here (perhaps at debug level) using the logger.